misc/kafka-util/scripts/print_archive.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+#
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Script to read all messages in an arrow archive of a kafka topic and print the python objects to
+# stdout
+
+import argparse
+import io
+import json
+import logging
+import pathlib
+import pprint
+import sys
+import typing
+
+import avro.io  # type: ignore
+import avro.schema  # type: ignore
+import pyarrow  # type: ignore
+import pyarrow.fs  # type: ignore
+
+# Setup basic formatting for logging output
+logging.basicConfig(format="%(asctime)s %(levelname)s %(name)s %(message)s")
+log = logging.getLogger("print.records")
+log.setLevel(logging.INFO)
```

Comment:
TIOLI: you can pass level to basicconfig:

```suggestion
logging.basicConfig(format="%(asctime)s %(levelname)s %(name)s %(message)s", level=logging.INFO)
log = logging.getLogger("print.records")
```

Reply:
Done!